### DISCUSSION TITLE
support clp LP solver?

### DISCUSSION THREAD

**Discussion by C8504** [google/or-tools#2776] [CP-SAT questions] [answered]
Hi Experts,

Could I know if we have any plan to support external LP solver inside CP-SAT solving? The default LP solver is GLOP as I know.

Thanks.

John

**Comment by lperron** [comment, marked as answer]
No. There is no added value of using CLP inside CP-SAT instead of GLOP. And CLP is not actively developed, which is not the case for GLOP.

**Reply by C8504** [reply]
exactly, thanks
